### PR TITLE
Add custom wireless profile, ssh key, 6 locations

### PR DESCRIPTION
**Pull request by ZuquetuPlexis**
1. Add custom wireless profile for HaCraFu e.V.
2. Add ssh key from HaCraFu-Member @ZuquetuPlexis
3. Add 6 locations from local community

### CODE REVIEW

File: locations/hacrafu-fiocchi.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+  # MGMT                      # create a management vlan in which we can reach every device on this site for maintenance
+  - vid: 42
+    role: mgmt
+    prefix: 10.31.202.95/32
+    gateway: 1
+    dns: 1
+    ipv6_subprefix: 1
+    assignments:
+      hacrafu-fiocchi-core: 1
```
Comment (Noki) [inline locations/hacrafu-fiocchi.yml:67]: From looking at the map it looks like you have an directional antenna attached to this device pointing towards ditalini, right? Usually you would use a larger management network here and add directional antennas or switches so the management interface can be accessed.

File: locations/hacrafu-barbine.yml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  - 'freifunk@hacrafu.de'
+
+hosts:
+
+  - hostname: hacrafu-barbine-core
+    role: corerouter
+    model: "tplink_tl_wdr4900_v1"
+    wireless_profile: freifunk_hacrafu
+
+#ipv6_prefix: '2001:bf7:850:b00::/56'
```
[the remark this answers is not in the document]
Comment (ZuquetuPlexis) [reply locations/hacrafu-barbine.yml:19]: barbine is one of our "oldest" nodes.I think we did not get a ipv6 address for it. Therefore it's missing in the config.

File: locations/hacrafu-fiocchi.yml (hunk 1 above)
Comment (ZuquetuPlexis) [reply locations/hacrafu-fiocchi.yml:67]: The long link to Ditalini is by Acciedent. It's caused by our configuration process.

File: group_vars/all/ssh-keys.yml [hunk 3]
```diff
@@ -28,3 +28,5 @@ all__ssh_keys__to_merge:
     key: ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIOaljQHr4Nj6veeDi51BZjy/a3WXPmH8SLePB/s/owNy noxil@systemli.org
   - comment: rtznprmpftl
     key: ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIHZ5H1Y3o30QvU3ohtXiWZUgw9fIGxvW35STG7vICF8O localghost
+  - comment: tom
+    key: ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAICIpPZouLOf+1WT9ylMa/9mX1dhLTy8W07Q8G5w7KKNz freifunk@hacrafu.de
```
[the remark this answers is not in the document]
Comment (ZuquetuPlexis) [reply group_vars/all/ssh-keys.yml:32]: We decided to only use hacrafu member keys for normal hardware deployments. Therefore only my key is currently present in the files.

[the remark this answers is not in the document]
Comment (Noki) [reply group_vars/all/ssh-keys.yml:32]: this is fine, but adding your key to group_vars/all/ssh-keys.yml would mean that you get access to all bbb-config nodes that use the default keys.

File: locations/hacrafu-barbine.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (Noki) [reply locations/hacrafu-barbine.yml:19]: just request additional IPs via https://config.berlin.freifunk.net/expert/form or register completely new ips and return the old ones...

File: locations/hacrafu-fiocchi.yml [hunk 4]
```diff
@@ -0,0 +1,72 @@
+---
+
+location: hacrafu-fiocchi
+location_nice: Apothekerteich-Fiocchi-Config
+latitude: 52.52320
+longitude: 13.77234
+contact_nickname: 'HaCraFu'
+contacts:
+  - 'freifunk@hacrafu.de'
```
[the remark this answers is not in the document]
Comment (ZuquetuPlexis) [reply locations/hacrafu-fiocchi.yml:9]: Like this?:

contact_name: 'Hacken Craften Funken e.V.''
contact_nickname: 'HaCraFu'
contacts:
  - 'freifunk@hacrafu.de'

**Comment by ZuquetuPlexis** [comment]
# ToDo

- [ ] rebase merge-commits
  - into which commit?
- [ ] create commit for new wireless profile
- [ ] create commit for new ssh key?
  - according to @Noki it might not be needed
- [ ] create commits for every new location

**Pull request closed without merging**